static/svg/luva-inkscape.svg

```diff
@@ -0,0 +1,134 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!-- Created with Inkscape (http://www.inkscape.org/) -->
```

Comment:
I could probably remove that comment. I had some issues when running these SVG files though an optimizer because somehow my CSS styling of them stopped working. If you could shed any light on that, I'd be happy :)